src/rust/engine/fs/src/lib.rs

```diff
@@ -463,25 +463,26 @@ pub struct PathGlobs {
 
 impl PathGlobs {
   pub fn create(
-    include: &[String],
-    exclude: &[String],
+    globs: &[String],
     strict_match_behavior: StrictGlobMatching,
     conjunction: GlobExpansionConjunction,
   ) -> Result<PathGlobs, String> {
-    let include = PathGlob::spread_filespecs(include)?;
-    Self::create_with_globs_and_match_behavior(include, exclude, strict_match_behavior, conjunction)
-  }
-
-  fn create_with_globs_and_match_behavior(
-    include: Vec<PathGlobIncludeEntry>,
-    exclude: &[String],
-    strict_match_behavior: StrictGlobMatching,
-    conjunction: GlobExpansionConjunction,
-  ) -> Result<PathGlobs, String> {
```

Comment:
Removed. It only had two call sites and wasn't pulling its weight anymore now that we unified `include` and `exclude`.

Reply:
Yes! I believe this was an artifact of the original change that introduced glob matching error messages, which underwent seemingly a million iterations (thanks so much for that @stuhood!!!) partially because I wrote it while undergoing a temporarily discombobulating medication regimen for a herniated disk. Please feel free to make such changes in the future -- it's awesome to see the compounding effects of simplifying syntax in a change like this (it's like how compilers can optimize functions away so much they can then be inlined).